embassy-stm32-wpan/src/sub/mm.rs

```diff
@@ -58,6 +58,7 @@ impl MemoryManager {
             })
             .await;
 
+            trace!("EMPTY MM");
```

Comment:
This should probably be removed.

Reply:
Removed altogether. It was a test that didn't need to be included.

Reply:
Well, that makes my life easier as well, since I did my work before you pushed it.